decam_process.py

```diff
@@ -207,158 +246,307 @@ def doIngest(repo, datafiles):
     config = IngestConfig()
     config.parse.retarget(DecamParseTask)
     # create an instance of the decam ingest task
-    ingestTask = ingest.DecamIngestTask(config=config)
+    IngestTask = ingest.DecamIngestTask(config=config)
```

Comment:
Probably because the story of my life is being confused about what is a class and what is an instance... I'll change it back :)